packages/p2p/src/utils/chat-message.ts

```diff
@@ -0,0 +1,77 @@
+import { FileMessage, UserMessage } from '@sendbird/chat/message';
+
+type TChatMessageArgs = {
+    created_at: number;
+    channel_url: string;
+    id: string;
+    file_type?: string;
+    message?: string;
+    message_type: string;
+    name?: string;
+    sender_user_id: string;
+    status?: number;
+    url?: string;
+};
+
+export default class ChatMessage {
```

Comment:
```suggestion
export default class ChatMessage implements TChatMessageArgs {
```

Reply:
just a suggestion. As we are not going to use this interface anywhere and declare class only once here, you can ignore my comments I think :)

Reply:
@yauheni-deriv, This is only used here for now, so I decided not to `export` it. Maybe If someone ever needs it in the future, they can `export` it? 🤔